v4/weaviate/auth/auth.go

```diff
@@ -0,0 +1,119 @@
+package auth
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/semi-technologies/weaviate-go-client/v4/weaviate/connection"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/clientcredentials"
+	"log"
+	"net/http"
+	"strings"
+)
+
+const OidcConfigURL = "/.well-known/openid-configuration"
```

Comment:
is this const used outside the package? if not then it can be defined as a private variable, with a lower letter